core/src/main/java/io/crate/module/CrateCoreModule.java

```diff
@@ -46,6 +47,10 @@
     public CrateCoreModule(Settings settings, IndexEventListenerProxy indexEventListenerProxy) {
         logger = Loggers.getLogger(getClass().getPackage().getName(), settings);
         this.indexEventListenerProxy = indexEventListenerProxy;
+        if (settings.getAsBoolean("license.enterprise", false) &&
+            "".equals(settings.get("license.ident",""))){
+            logger.warn("You are using an unlicensed version of CrateDB Enterprise Edition");
```

Comment:
The cluster check description is more verbose - should we extend the warning here as well?